src/app/app.component.scss

```diff
@@ -15,10 +15,28 @@
 
 .presentation-wrapper {
     float: left;
-    width: 65%;
+    width: 50%;
+    
 }
 
 .upcoming-events-wrapper {
     float: right;
-    width: 32%;
+    width: 50%;
+}
+
+ p {
+    text-align: center;
+    font-size: 25px;
+    margin-bottom: 50px;
+    font-family: 'Fredoka One', cursive;
+}
+
+.left-title {
+    margin-left: 150px;

```

Comment:
Lets either center the title or left justify it.   Using a left margin doesn't update in responsive design very well.
Use the Chrome dev tools and toggle the device toolbar, then switch to a phone, say iPhone 5 and see the layout.
We are targeting phones as a primary use case as well as desktop browsing experience.

Reply:
Gotcha.  Let me play around with the wrappers because if we center on the left wrapper at 50% width it moves to far to the right.

Reply:
Sounds good.   Just keep responsive design in mind. :P